From: Colin Ian King <colin.king@xxxxxxxxxxxxx>

building with gcc 6 I hit a build failure:

  CC      drivers/staging/wilc1000/wilc_spi.o
drivers/staging/wilc1000/wilc_spi.c:123:34: error: storage size of
 ‘wilc1000_spi_ops’ isn’t known
 static const struct wilc1000_ops wilc1000_spi_ops;
                                  ^~~~~~~~~~~~~~~~
drivers/staging/wilc1000/wilc_spi.c:123:34: warning: ‘wilc1000_spi_ops’
  defined but not used [-Wunused-const-variable]

Since wilc1000_spi_ops is not used, remove it and the driver builds.

Signed-off-by: Colin Ian King <colin.king@xxxxxxxxxxxxx>
---
 drivers/staging/wilc1000/wilc_spi.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/wilc1000/wilc_spi.c b/drivers/staging/wilc1000/wilc_spi.c
index 86de50c..b3d6541 100644
--- a/drivers/staging/wilc1000/wilc_spi.c
+++ b/drivers/staging/wilc1000/wilc_spi.c
@@ -120,8 +120,6 @@ static u8 crc7(u8 crc, const u8 *buffer, u32 len)
 
 #define USE_SPI_DMA     0
 
-static const struct wilc1000_ops wilc1000_spi_ops;
-
 static int wilc_bus_probe(struct spi_device *spi)
 {
 	int ret, gpio;
-- 
2.7.0
